securesystemslib/signer/_crypto_signer.py

```diff
@@ -108,24 +108,32 @@ class CryptoSigner(Signer):
 
     A CryptoSigner can be created from:
 
-        a. private key file -- ``Signer.from_priv_key_uri()``
+        a. private key file -- see ``Signer.from_priv_key_uri()``
 
-          URI has the format "file:<PATH>?encrypted=[true|false]", where
-          PATH is the path to a file with private key data in a standard
-          PEM/PKCS8 format.
+          This is the generic (not CryptoSigner specific) way to
+          create a signer: use this when you already have a private
+          key  (and a private key URI) you can use.
 
-          A related public key must be passed.
+          URI has the format "file2:<PATH>", where PATH is a filesystem
+          path to the private key file. If CRYPTO_SIGNER_PATH_PREFIX
+          environment variable set, the private key will be read from
+          ``CRYPTO_SIGNER_PATH_PREFIX + <SEPARATOR> + PATH``. The
+          purpose of this is to allow PATH to only encode an identifier
+          (e.g. filename) while allowing the signing system to store
+          the private keys where ever it wants at runtime.
 
-          If  ``encrypted=true``, the optional secrets handler is expected to
-          return a decryption password.
+        b. newly generated key pair -- see ``CryptoSigner.generate_*()``
 
-        b. newly generated key pair -- ``CryptoSigner.generate_*()``
+          Use this when you need a brand new private key pair.
 
         c. existing pyca/cryptography private key object -- ``CryptoSigner()``
 
+          Use this if you need a brand new private key pair and option
+          b is not flexible enough for your case.
     """
 
-    FILE_URI_SCHEME = "file"
+    SCHEME = "file2"
+    PREFIX_ENV_VAR = "CRYPTO_SIGNER_PATH_PREFIX"
```

Comment:
the environment variable name could have _securesystemslib_ in it but I suppose it's long enough already

Reply:
I think it's fine. An application can subclass CryptoSigner and override `PREFIX_ENV_VAR`.